README.md

```diff
@@ -19,4 +19,8 @@ Contributions welcome!
 - ตอบแทนบุญคุณแผ่นดิน
 - ควายแดง
 - สวะส้ม
+- ชั่ยๆ
+- ไอแม้ว
+- เลวจริงๆ
```

Comment:
This line too, cannot classify good enough eg.
- เรื่องหมอนี่อยากร้องไห้มากๆ หมอไม่ได้ขึ้นไปพูดบนเวทีด้วยซ้ำ เขาไปช่วยดูแลประชาชนอ่ะ สารเลวจริงๆ